### PR TITLE
Awesome books

### PR DESCRIPTION
**Pull request by peterboro**
In this pull request I implemented the following: 
- Create a new directory and initialize it with git.
- Use npm init -y command to create package.json file.
- Create the entry point for your JavaScript code called index.js
- Create index.css file for your styles.
- Create the main index.html file. Link your index.js and index.css files there.
- Create a directory called modules.

- Re-create your Awesome books app following the rules:
- Use ES6 syntax.
- Divide your code into modules. Save each module in a separate file in the modules dir. Import modules in the index.js file. For this exercise - the more modules the better!
- Keep all funcionalities of the app without errors.
- Refactor your methods and functions to arrow functions.
- Use let and const in a correct way.
- Display the current date below the navigation bar.

- For this requirement you should use a library [Luxon](https://moment.github.io/luxon/#/install?id=es6)
- Download the script and save it to your project.
- Import it as a module.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  }
+});
+
+document.querySelector('#list-book').addEventListener('click', (e) => {
+  task.deleteBook(e.target);
+  Store.removeBook(e.target.parentElement.previousElementSibling.textContent);
+  task.showAlert('Book Removed');
+});
+
+function displaySection(id) {
```
Comment (uwadonat) [inline index.js]: - Let's use arraw functions for all your functions as requested in project requirements

File: index.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    case 'contact':
+      sectionId = 'contact-section';
+      break;
+    default:
+      sectionId = '';
+  }
+  return sectionId;
+}
+
+// window.addEventListener('load', () => {
```
Comment (uwadonat) [inline index.js]: - It's better to remove all commented lines to make the project more clean

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @peterboro,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your commit and pull request messages are very descriptive ✔️
- No linter errors ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by muneebulrehman — APPROVED** [review]
Hi @peterboro , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 2 above)
Comment (peterboro) [reply index.js]: Thank you for the review.

**Comment by peterboro** [comment]
Thank you for the review and approval.

**Pull request merged**